test/cbmc/proofs/run-cbmc-proofs.py

```diff
@@ -0,0 +1,233 @@
+#!/usr/bin/env python3
```

Comment:
This will soon go into aws-templates-for-cbmc-proofs right?

Reply:
Not completely sure. It would be good if it did, but on the other hand there are slight differences between the projects.

You're right though, it really would be better if we could take the common parts of it and put it in the starter kit.

Reply:
Have we not made minor changes to this code for every project?

Reply:
Yes. That's because unlike with Litani itself, I didn't really have a good idea of what the commonalities/differences would need to be for the run-script. So I put a fresh one into each project and figured that we could consolidate them later if it turned out that there was lots in common.

For these C-SDK project, and even s2n, the scripts have turned out to be almost identical. So it would be worth re-visiting whether the script should live in the starter kit at some point.